docs/src/modules/ROOT/pages/quickstart/quarkus/quarkus-quickstart.adoc

```diff
@@ -64,9 +64,14 @@ To complete this guide, you need:
 
 == The build file and the dependencies
 
-Use https://code.quarkus.io/[code.quarkus.io] to generate an application
+Use https://code.quarkus.io/?a=timefold-solver-quickstart&j=17&e=resteasy&e=resteasy-jackson&e=ai.timefold.solver%3Atimefold-solver-quarkus-jackson&e=ai.timefold.solver%3Atimefold-solver-quarkus[code.quarkus.io] to generate an application
```

Comment:
```suggestion
Use https://code.quarkus.io/?a=timefold-solver-quickstart&j=21&e=resteasy&e=resteasy-jackson&e=ai.timefold.solver%3Atimefold-solver-quarkus-jackson&e=ai.timefold.solver%3Atimefold-solver-quarkus[code.quarkus.io] to generate an application
```

Maybe we want to nudge people towards the latest LTS?

Reply:
I will pick this up in another pull request, will want to make sure this actually doesn't cause any issues.